pyccel/parser/utilities.py

```diff
@@ -91,7 +93,7 @@ def accelerator_statement(stmt, accel):
     """
     assert(accel in ['omp', 'acc'])
 
-    if not isinstance(stmt, CommentNode): None
+    if not isinstance(stmt, CommentLine): None
```

Comment:
These statements do nothing. Are they supposed to be:
`if not isinstance(stmt, CommentLine): `return None?

Reply:
they should return something